xbmc/media/MediaType.h

```diff
@@ -37,6 +37,7 @@ using MediaType = std::string;
 #define MediaTypeTvShow           "tvshow"
 #define MediaTypeSeason           "season"
 #define MediaTypeEpisode          "episode"
+#define MediaTypeShoutcast        "shoutcast"
```

Comment:
"shoutcast" shouldnt be a mediatype. Mediatypes should describe what kind of media it is, not which protocol it is.